k2/csrc/top_sort_test.cu

```diff
@@ -194,4 +195,61 @@ TEST(TopSort, RandomVectorOfFsas) {
   }
 }
 
+// another random test which uses IsRandEquivalent to check the result
+TEST(TopSort, RandomVectorOfFsas1) {
+  ContextPtr cpu = GetCpuContext();
+  for (int32_t n = 0; n != 0; ++n) {
```

Comment:
Ignore this, use `n!=0` on purpose as there are still bugs in GPU version of TopSort, will continue debugging it.